textbook/12/empirical-distributions.ipynb

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
<p>This looks like a typo resulting in a broken image:</p><p> </p><pre class="ql-syntax" spellcheck="false">![Chapter 11](../11/Probability.ipynb)
</pre><p> </p><p>Rather than say a link:</p><p> </p><pre class="ql-syntax" spellcheck="false">[Chapter 11](../11/Probability.html)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chicago-cdac/textbook-datascience-1/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>You are absolutely right that the ! is a typeo. The file should be .html instead of .ipynb? Will a rendered html file be in that folder in the built version of the textbook? Asking because I think I made this mistake other places if so...</p>

Reply:
Yes, you're right. The ambiguity in authoring such things is honestly frustrating.

There are generally three ways about it:

* Write proper HTML, which will be passed through as-is, and work in the textbook.
* Write Markdown, which should be fixed up before conversion to HTML.
* Use a Jupyter Book (Sphinx) helper, which will ostensibly make things easier, (but this markup may be even less natural).

[Here's documentation I found](https://jupyterbook.org/en/stable/content/references.html#reference-content-files).

And so, any of the following would work:

* `<a href="../11/Probability.html">Chapter 11</a>`
* `[Chapter 11](../11/Probability.ipynb)`
* `[](../11/Probability.ipynb)`
* `[Chapter 11](../11/Probability)`
* `[](../11/Probability)`
* ``{doc}`Chapter 11 <../11/Probability>` ``
* ``{doc}`../11/Probability` ``

That is, all references are relative, and:

* HTML should presume the full final document path (with `.html` suffix)
* Markdown may either refer to the actual file name (including `.ipynb` suffix) or omit the suffix entirely
* Sphinx-style references may not include the file suffix
* Markdown and Sphinx references may omit the link text, in which case it will be derived from the referenced document's full header, (in this case `Probability: Mathematical/Theoretical and Computational Approaches`).

So! Yes, in short, this would do the trick, (among other options)…:

```
[Chapter 11](../11/Probability.ipynb)
```

Reply:
Goodness, so confusing! Thank you for clarifying! I have pushed the change, so hopefully it works now! Please let me know if other changes are needed

Reply:
Thanks! Looks good to me. So this is ready to merge?

Reply:
I think so